### ISSUE TITLE
Show Your Work: Improved Reporting of Experimental Results

### ISSUE THREAD

**Issue by Theodb** [Theodb/PhD-research-papers#4]
## TL;DR
New metrics to evaluate models: accuracy of the dev depending of the number of hyperparameters
* find the maximum expectation of the dev performance 

<img width="553" alt="Capture d’écran 2020-10-02 à 13 54 19" src="https://user-images.githubusercontent.com/55734235/94920536-cc1ddc00-04b6-11eb-9fcd-5385fc25c44c.png">

Show difference between teams models - budget (nb of hyperparameters that could tested).
Effectively here the model with the worst reported accuracy is in fact the best model with a bigger budget.
<img width="279" alt="Capture d’écran 2020-10-02 à 13 58 45" src="https://user-images.githubusercontent.com/55734235/94920861-6847e300-04b7-11eb-86dd-8c1fcdaf18b9.png">

### Article Link
https://www.aclweb.org/anthology/D19-1224.pdf
### Author
Jesse Dodge et al.
### Conference
2019 Conference on Empirical Methods in Natural Language Processing
and the 9th International Joint Conference on Natural Language Processing
## Key Takeaways
* 
* 

## Useful Code Snippets
```python
#Add your codes here
import numpy as np
import pandas as pd

```

## Useful Tools
* 
*
